src/algebra/order/with_zero.lean

```diff
@@ -171,6 +171,24 @@ begin
   simpa [inv_mul_cancel_left₀ ha, inv_mul_cancel_left₀ (ne_of_gt hc)] using this,
 end
 
+lemma with_zero.div_le_div_right (hc : c ≠ 0) : a/c ≤ b/c ↔ a ≤ b :=
+begin
+  rw [div_eq_mul_inv _ _, div_eq_mul_inv _ _],
+  split; intro hab,
+  { rw [← mul_one a, ← mul_one b, ← inv_mul_cancel hc, ← mul_assoc, ← mul_assoc],
+    exact mul_le_mul_right' hab _, },
+  { exact mul_le_mul_right' hab _, }
+end
+
+lemma with_zero.div_le_iff (hb : b ≠ 0) : a/b ≤ c ↔ a ≤ c*b :=
+begin
+  conv_lhs {rw ← mul_one c, rw ← div_self hb, rw ← mul_div_assoc},
+  exact with_zero.div_le_div_right hb,
+end
+
+lemma with_zero.mul_le_one (ha : a ≤ 1) (hb : b ≤ 1) : a*b ≤ 1 :=
+by { rw ← mul_one (1 : α), exact mul_le_mul' ha hb }
+
```

Comment:
This lemma already exists, it's `mul_le_one'` from `order/monoid_lemmas.lean`

```suggestion
```

Reply:
Ok, I'll remove it.